Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -674,9 +674,18 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 		/// <param name="id">Network id of the gameobject</param>
         public GameObject GetGameObjectByNetworkId(uint id)
         {
-			NetworkObject foundNetworkObject;
-			Networker?.NetworkObjects?.TryGetValue(id, out foundNetworkObject);
-            return ((NetworkBehavior)foundNetworkObject?.AttachedBehavior?).gameObject?;
+            if (Networker == null ) //Only check Networker, as NetworkObjects are always initiliased.
+            {
+                Debug.LogWarning("Networker is null. Check if initiliased");
+                return null;
+            }
+            NetworkObject foundNetworkObject = null;
```

Comment:
Minor thing. Could you add an empty line above this please?

Reply:
Above the NetworkObject declaration & return?

Reply:
yes please. Between the `}` and the line with `NetworkObject`